Sources/SwiftDocC/Model/Rendering/DocumentationContentRenderer.swift

```diff
@@ -219,9 +219,15 @@ public class DocumentationContentRenderer {
         // Verify that if current platforms are in beta, they match the introduced version of the symbol
         for availability in symbolAvailability {
             // If not available on this platform, skip to next platform.
-            guard !availability.isUnconditionallyUnavailable, let introduced = availability.introducedVersion else {
+            guard !availability.isUnconditionallyUnavailable else {
                 continue
             }
+
+            // If the availability item does not have an introduced version,
+            // the symbol can't be marked as beta.
```

Comment:
How about updating this comment to something like this do describe the broader expectation about when a symbol is considered "in beta" and why this code follows those expectations. With the current comment I thought that the early exit could be a bug at first.

```suggestion
            // Symbols are only considered "in beta" if they are in beta for all platforms that they are available for.
            // If the symbol doesn't have an introduced version for one of those platforms, we don't consider it "in beta".
```

Reply:
Resolved [here](https://github.com/swiftlang/swift-docc/pull/1086/commits/c5458a77109ad6c8f263c4268bdb5586701f5a9e)